src/main/io/rc_controls.c

```diff
@@ -392,260 +400,126 @@ void applyStepAdjustment(controlRateConfig_t *controlRateConfig, uint8_t adjustm
     }
     switch(adjustmentFunction) {
```

Comment:
some way to decide what area to adjust will be necessary .. should be quite simple, only 2 starting pointers are used now